plugs/directive/query_directive.ts

```
@@ -17,10 +17,10 @@ export async function queryDirectiveRenderer(
   const results = await events.dispatchEvent(
     `query:${parsedQuery.table}`,
     { query: parsedQuery, pageName: pageName },
-    30 * 1000,
+    30 * 1000
   );
   if (results.length === 0) {
-    return "";
+    return "No results";
```

Have you tried if this actually works? I haven’t but I’m pretty sure it doesn’t. The case of `results.length === 0` happens when you’re querying a query source that doesn’t exist (this should probably return an error message.

Where you’ll want to make changes is a few lines down (line 29, the one with `jsonToMdTable`, if that `results[0]` is an empty array, then you show `No results`.